<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget, QGraphicsView {
background: #1529bd;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="field">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>352</width>
      <height>702</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: teal;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="fieldAdditional">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>200</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: teal;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>530</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border: 2px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="startTetris">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
     background: #00d200;
     border-style: outset;
     border-width: 2px;
     border-color: beige;
 }

QPushButton:hover {
	background: #119711;
}

QPushButton:pressed {
	background: #186e18;
}</string>
     </property>
     <property name="text">
      <string>TETRIS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startSnake">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
     background: #00d200;
     border-style: outset;
     border-width: 2px;
     border-color: beige;
 }

QPushButton:hover {
	background: #119711;
}

QPushButton:pressed {
	background: #186e18;
}</string>
     </property>
     <property name="text">
      <string>SNAKE</string>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="hScore">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="score">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>330</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="level">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>410</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>678</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background: #d22600;
     border-style: outset;
     border-width: 2px;
     border-color: beige;
 }

QPushButton:hover {
	background: #bd3315;
}

QPushButton:pressed {
	background: #972911;
}</string>
    </property>
    <property name="text">
     <string>QUIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>470</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background: #c0c0c0;
     border-style: outset;
     border-width: 2px;
     border-color: beige;
 }

QPushButton:hover {
	background: #9a9a9a;
}

QPushButton:pressed {
	background: #7b7b7b;
}</string>
    </property>
    <property name="text">
     <string>PAUSE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>389</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
padding-left: 5px;
border-top-left-radius: 10px</string>
    </property>
    <property name="text">
     <string>LEVEL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>310</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
padding-left: 5px;
border-top-left-radius: 10px</string>
    </property>
    <property name="text">
     <string>SCORE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
padding-left: 5px;
border-top-left-radius: 10px</string>
    </property>
    <property name="text">
     <string>HIGH SCORE</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameOver">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>TO START NEW GAME 
PRESS ENTER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
